helpers/authentication.go

```diff
@@ -2,12 +2,19 @@ package helpers
 
 import (
 	"encoding/json"
+
 	"go-google-scraper-challenge/helpers/log"
-	"go-google-scraper-challenge/lib/api/v1/serializers"
 
 	"golang.org/x/crypto/bcrypt"
 )
 
+type AuthenticationToken struct {
```

Comment:
Moved this from serializer to avoid import cycle error on run time

Reply:
Could it be possible to move this to `models` instead? 🤔

Reply:
I give it a try but nope, it'll still face the import cycle error
![Screen Shot 2565-07-11 at 10 00 43](https://user-images.githubusercontent.com/1772999/178180813-27b9bd58-668a-49c7-a7a0-104281060f4f.png)